Project.toml

```
@@ -30,6 +30,7 @@ StrLiterals = "68059f60-971f-57ff-a2d0-18e7de9ccc84"
 ThreadsX = "ac1d9e8a-700a-412c-b207-f0111f4b6c0d"
 
 [compat]
+RandomNumbers = "~1.4"
```

```suggestion
RandomNumbers = "^1.5.1"
```
The problem in `RandomNumbers` is fixed now.

---

Ah, great!  I thought that it looked like a bug on their end.

---

Or maybe it is not fixed? Tests seem to fail with `1.5.1`, now with a `StackOverFlowError`, but still during seeding of the RNG.
See als #97